.github/workflows/release-module-npm.yml

```diff
@@ -0,0 +1,60 @@
+name: Release Module NPM
+on:
+  workflow_call:
+    inputs:
+      env_vars:
+        required: false
+        type: string
+        default: '{}'
+      registry_url:
+        required: false
+        type: string
+        default: 'https://registry.npmjs.org'
+      registry_scope:
+        required: false
+        type: string
+        default: '@digicatapult'
+      npm_build:
+        required: false
+        type: boolean
+        default: false
+      package_access:
+        required: false
+        type: string
+        default: 'public'
+    secrets:
+      AUTH_TOKEN:
```

Comment:
maybe `NPM_AUTH_TOKEN` to be more explicit?

Reply:
Well no, because it could be any registry?

Reply:
`REGISTRY_AUTH_TOKEN`?